ios/FluentUI.Demo/FluentUI.Demo/Demos/ButtonDemoController.swift

```diff
@@ -50,4 +79,15 @@ extension ButtonStyle {
             return "Borderless"
         }
     }
+
+    var image: UIImage? {
+        switch self {
+        case .primaryFilled, .primaryOutline:
+            return UIImage(named: "ic_fluent_circle_24_regular")!
```

Comment:
totally find if the imageset name is more user "friendly" i only cared about the actualy pdf file name